packages/avatar/stories/avatar.stories.tsx

```diff
@@ -8,15 +8,8 @@ import Avatar, { SkeletonType } from '../src/Avatar';
 
 export default {
   title: 'Components/Avatar',
-  parameters: {
-    docs: {
-      // page: README,
-      description: {
-        component: 'Availity user avatar component.',
```

Comment:
I would still leave in descriptions. It can be done through params like this, or you could move it into documentation comments above the default export.